### ISSUE TITLE
Create a guide/example for file upload or multi-part form data

### ISSUE THREAD

**Issue by ekryski** [feathersjs/feathers-docs#111]
This could include using https://github.com/expressjs/multer (or any other Express middleware for file upload) or https://github.com/feathersjs/feathers-blob, preferably the latter.

**Comment by queckezz** [comment]
I haven't been able to get a form-data example working with `multer` and `feathers`. It requires both `req` and `res` which is not available in a feathers pre- or posthook. It also doesn't make sense as a normal middleware since it would then also parse `create`, `find` etc. Can you guys give me any directions?

This is how I do it without feathers atm:
```js
app.post('/storage', upload.single('file'), (req, res, next) => {
  res.send('Successfully uploaded')
})
```

**Comment by daffl** [comment]
Closed via #171
